Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'pygments.rb', '~> 0.6.0'
-gem 'redcarpet', '~> 3.1'
+gem 'redcarpet', '~> 3.2.3'
```

Comment:
This unnecessarily locks the user to using 3.2.x, instead of 3.x. I believe you can declare this as `~> 3.2, >= 3.2.3`.